forms.py

```diff
@@ -5,11 +5,18 @@
 from flask_wtf.file import FileField, FileAllowed, FileRequired
 
 class TableForm(FlaskForm):
-    table_number = IntegerField( validators=[InputRequired(), NumberRange(1,100)])
-    seats = IntegerField(validators=[InputRequired(), NumberRange(1,25)])
+    table_number = IntegerField("Assign Table Number", validators=[InputRequired(), NumberRange(1,100)])
+    seats = IntegerField("Number Of Seats", validators=[InputRequired(), NumberRange(1,25)])
+    '''
+    instead of submitting coordinates, can we not set the default coordinates to (0,0)
```

Comment:
Not sure about this,(0,0) won't be visible and if it's put in the middle of the page could be on top of another table.